controllers/common.go

```diff
@@ -61,6 +67,7 @@ func (r *Reconciler) EnsureLogsPVCExists(ctx context.Context, instance client.Ob
 					corev1.ResourceStorage: k8sresource.MustParse("1Gi"),
 				},
 			},
+			StorageClassName: &StorageClassName,
```

Comment:
Did you considered adding the vole claim parameters as struct to the parameters ?